Timezones/Timezones.plugin.js

```diff
@@ -132,37 +132,41 @@ module.exports = (() => {
                   const TextInput = Webpack.getModule((m) => m?.Sizes?.MINI && m?.defaultProps?.type === "text", {
                       searchExports: true,
                   });
-                  const Markdown = Webpack.getModule((m) => m.Z?.rules && m.Z?.defaultProps?.parser).Z;
+                  const Markdown = Webpack.getModule((m) => m.default?.rules && m.default?.defaultProps?.parser).default
 
                   return class Timezones extends Plugin {
                       async onStart() {
                           injectCSS("Timezones-Styles", Styles);
 
-                          const ProfileBanner = Webpack.getModule((m) => m.Z?.toString().includes("e.displayProfile") && m.Z?.toString().includes("e.user"));
-                          const MessageHeader = Webpack.getModule((m) => m.Z?.toString().includes("userOverride") && m.Z?.toString().includes("withMentionPrefix"));
-                          const Tooltip = BdApi.Components.Tooltip;
+                          const ProfileBanner = Webpack.getModule(x=>x.default && x.default.toString().includes("y.GifAutoPlay")) //Webpack.getModule(x=>x.default && x.default.toString().includes("c.AvatarDecorationBorderSizes"))
```

Comment:
Why the comment?

Reply:
I added the comment to make sure to keep it stored just incase "y.GifAutoPlay" wasnt right